recipes-security/acms/acms.bb

```diff
@@ -3,63 +3,42 @@ HOMEPAGE = "https://software.intel.com/en-us/articles/intel-trusted-execution-te
 BUGTRACKER = "https://software.intel.com/en-us/forums/intel-trusted-execution-technology-intel-txt"
 SECTION = "bootloaders"
 LICENSE = "Intel-ACMs"
+FILESEXTRAPATHS ?= "${THISDIR}/acms"
 LIC_FILES_CHKSUM = " \
-    file://GM45_GS45_PM45-SINIT_51/license.txt;md5=60d123634e0b94f8c425003389e64bda \
-    file://Q45_Q43-SINIT_51/license.txt;md5=60d123634e0b94f8c425003389e64bda \
-    file://Q35-SINIT_51/license.txt;md5=60d123634e0b94f8c425003389e64bda \
-    file://i5_i7_DUAL-SINIT_51/license.txt;md5=60d123634e0b94f8c425003389e64bda \
-    file://i7_QUAD-SINIT_51/license.txt;md5=60d123634e0b94f8c425003389e64bda \
-    file://license.txt;md5=203b6b806e49ca139abdf6706024c871 \
-    file://3rd_gen_i5_i7-SINIT_67/license.txt;md5=a879c484244808a2202d65166a2f3f72 \
-    file://4th_gen_i5_i7-SINIT_75/license.txt;md5=a879c484244808a2202d65166a2f3f72 \
-    file://5th_gen_i5_i7-SINIT_79/license.txt;md5=68248a22232ba4fd23010e9c65209406 \
-    file://6th_gen_i5_i7-SINIT_71/license.txt;md5=1d626a5a455fd0bff690b866040198c2 \
-    file://7th_gen_i5_i7-SINIT_74/license.txt;md5=4897766beede80773ddd162b2724a8bb \
-    file://8th_gen_i5_i7-SINIT_76/license.txt;md5=24983614c000aac0d87e5aea9d13b8be \
+    file://license.txt;md5=24983614c000aac0d87e5aea9d13b8be \
 "
 
 SRC_URI = " \
-    ${OPENXT_MIRROR}/GM45_GS45_PM45-SINIT_51.zip;name=gm45 \
-    ${OPENXT_MIRROR}/Q45_Q43-SINIT_51.zip;name=q45 \
-    ${OPENXT_MIRROR}/Q35-SINIT_51.zip;name=q35 \
-    ${OPENXT_MIRROR}/i5_i7_DUAL-SINIT_51.zip;name=i5 \
-    ${OPENXT_MIRROR}/i7_QUAD-SINIT_51.zip;name=i7 \
-    ${OPENXT_MIRROR}/Xeon-5600-3500-SINIT-v1.1.zip;name=xeon_5600 \
-    ${OPENXT_MIRROR}/Xeon-E7-8800-4800-2800-SINIT-v1.1.zip;name=xeon_e7 \
-    ${OPENXT_MIRROR}/3rd-gen-i5-i7-sinit-67.zip;name=ivb_snb \
-    ${OPENXT_MIRROR}/4th-gen-i5-i7-sinit-75.zip;name=hsw \
-    ${OPENXT_MIRROR}/5th_gen_i5_i7-SINIT_79.zip;name=bdw \
-    ${OPENXT_MIRROR}/6th_gen_i5_i7-SINIT_71.zip;name=skl \
-    ${OPENXT_MIRROR}/7th_gen_i5_i7-SINIT_74.zip;name=kbl \
-    ${OPENXT_MIRROR}/8th_gen_i5_i7-SINIT_76.zip;name=cfl \
+    file://license.txt \
+    ${OPENXT_MIRROR}/SNB_IVB_SINIT_20190708_PW.bin;downloadfilename=snb_ivb.bin;name=snb_ivb \
+    ${OPENXT_MIRROR}/BDW_SINIT_20190708_1.3.2_PW.bin;downloadfilename=bdw_hsw.bin;name=bdw_hsw \
+    ${OPENXT_MIRROR}/SKL_KBL_AML_SINIT_20211019_PRODUCTION_REL_NT_O1_1.10.0.bin;downloadfilename=skl_kbl_aml.bin;name=skl_kbl_aml \
+    ${OPENXT_MIRROR}/CFL_SINIT_20221220_PRODUCTION_REL_NT_O1_1.10.1_signed.bin;downloadfilename=cfl_wkl_cml.bin;name=cfl_wkl_cml \
+    ${OPENXT_MIRROR}/CML_S_SINIT_1_13_33_REL_NT_O1.PW_signed.bin;downloadfilename=cml_s.bin;name=cml_s \
+    ${OPENXT_MIRROR}/CMLSTGP_SINIT_v1_14_39_20211214_REL_NT_O1.PW_signed.bin;downloadfilename=cml_s_tgp.bin;name=cml_s_tgp \
+    ${OPENXT_MIRROR}/RKLS_SINIT_v1_14_46_20220818_REL_NT_O1.PW_signed.bin;downloadfilename=rkls.bin;name=rkls \
```

Comment:
RKLS failed to download - it's 2022081**9**.

630744_003 also has:
CML_RKL_S_SINIT_v1.13.33_REL_NT_O1.PW_signed.bin
CMLSTGP_SINIT_v1_14_46_20220819_REL_NT_O1.PW_signed.bin
which are not on the mirror or included here in SRC_URI.

Reply:
The three CML_* files are identical:
```
5670905f72d7415fad97eb02bd994ae75c15ceec  630744_002/CML_S_SINIT_1_13_33_REL_NT_O1.PW_signed.bin
5670905f72d7415fad97eb02bd994ae75c15ceec  630744_003/CML_RKL_S_SINIT_v1.13.33_REL_NT_O1.PW_signed.bin
5670905f72d7415fad97eb02bd994ae75c15ceec  630744_003/CML_S_SINIT_1_13_33_REL_NT_O1.PW_signed.bin
```

Reply:
My mistake on the 20220819. Pushing a fix for that. The change to v1_14_46 for CMLSTGP slipped past me, pushing a fix for that as well.  It'll also go up on the mirror.

Not sure why they have 2 versions of the same sinit, but the CML_RKL_S isn't on the 630744_ACM_PlatformList_003.pdf, only the CML_S_SINIT_1_13_33 is.  So I'm going to advocate for only listing that one in the `SRC_URI` and chalk up the inclusion of the other one to user error on Intel's part.